check-for-missing-checkstyle-rule.py

```diff
@@ -30,22 +30,32 @@ def check_output_discarding_stderr(*args, **kwargs):
         return subprocess.check_output(*args, stderr=devnull, **kwargs)
 
 
+def try_decode(s):
```

Comment:
`try_decode` does not seem to abstract the steps in a way that is meaningful - and rather just something which exists as a way to minimise repetition. What do you think?

Reply:
Is `check_output_discarding_stderr` the only place which uses `subprocess.check_output`? If yes, I would actually make sure it always returns string and we can remove `try_decode`:
```
def check_output_discarding_stderr(*args, **kwargs):
    with open(os.devnull, 'w') as devnull:
        out = subprocess.check_output(*args, stderr=devnull, **kwargs)

        # subprocess.check_output return 'str' on Python 2 but 'bytes' on Python 3.
        #Therefore, do an optional conversion to string if necessary
        out = type(out) == bytes ? s.decode() : s

        return out
```